plugins/nginx-vhosts/templates/nginx.conf.sigil

```diff
@@ -12,7 +12,7 @@ server {
   access_log  /var/log/nginx/{{ $.APP }}-access.log;
   error_log   /var/log/nginx/{{ $.APP }}-error.log;
 {{ if (and (eq $listen_port "80") ($.SSL_INUSE)) }}
-  return 301 https://$host:{{ $.NGINX_SSL_PORT }}$request_uri;
+  return 301 https://$host:{{ $.PROXY_SSL_PORT }}$request_uri;
```

Comment:
This *still* has to be set in [this map](https://github.com/dokku/dokku/pull/2823/files#diff-a3f91ae60990139f055466b7c67a446bR289). `sigil` doesn't expand the environment into templates.

And I would say we should support the old versions as well.